google/cloud/security/common/data_access/sql_queries/select_data.py

```diff
@@ -149,11 +149,24 @@
     WHERE project_number = {1};
 """
 
-
-SELECT_VIOLATIONS = """
+SELECT_POLICY_VIOLATIONS = """
     SELECT * FROM violations_{0};
 """
 
+SELECT_BIGQUERY_ACL_VIOLATIONS = """
+    SELECT * FROM violations_{0}
+    WHERE violation_type = 'BIGQUERY_VIOLATION';
```

Comment:
Please use %s instead of {1}, because of sql injections. We only use the format placeholders for tablenames.

https://stackoverflow.com/questions/7929364/python-best-practice-and-securest-to-connect-to-mysql-and-execute-queries